input/cmd_list.c

```diff
@@ -348,6 +352,11 @@ bool mp_input_is_repeatable_cmd(struct mp_cmd *cmd)
            (cmd->flags & MP_ALLOW_REPEAT);
 }
 
+bool mp_input_is_scalable_cmd(struct mp_cmd *cmd)
+{
+    return cmd->def && cmd->def->scalable;
+}
+
```

Comment:
Hmm, would that be an improvement? Like key-repeat, scalability is a property of the command definition rather than the mp_cmd instance.